README.md

```diff
@@ -1,25 +1,10 @@
-### Deploy a Kubernetes Cluster
+## Deploy a Kubernetes Cluster with Terraform
+This repository provides examples of provisioning Kubernetes clusters on major cloud providers using Terraform. These should be treated as illustrative examples, not production-ready infrastructure configuration.
 
-Pick the platform you want to deploy your cluster to, and follow the instructions in the README of the corresponding directory.
+### Get Started
+- Install [terraform](https://developer.hashicorp.com/terraform/downloads)
+- `cd` into the directory corresponding to your cloud provider (AWS, GCP, or Azure)
+- Follow the instructions in the corresponding README
+- Monitor the Terraform output and wait for your cluster to deploy
 
-## Managing Your Cluster with Kubectl
-Follow these steps after you've deployed your cluster to AWS or GCP. 
-
-To manage your cluster, you can use the `kubectl` [cli tool](https://kubernetes.io/docs/tasks/tools/), which you'll need to configure cluster access for.
-
-1. For both commands you'll need to set the env variables first:
-    ```bash
-    export $REGION=<the region code for your cluster>
-    export $CLUSTER_NAME=<the name of your cluster on GCP or AWS>
-    ```
-
-2. If you are using [`gcloud`](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl), configure `kubectl` by running:
-
-`gcloud container clusters --region=$REGION get-credentials $CLUSTER_NAME`
-
-3. If you are using [`aws`](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html), configure `kubectl` by running:
-
-`aws eks update-kubeconfig --region $REGION --name $CLUSTER_NAME`
-
-4. Run `kubectl config current-context` to see the current context
-5. (Optionally) Run `kubectl cluster-info` to see information about the current cluster
+That's it. Next up, [deploy Sourcegraph](https://docs.sourcegraph.com/admin/deploy/kubernetes).
```

Comment:
```suggest
Next, [deploy Sourcegraph](https://docs.sourcegraph.com/admin/deploy/kubernetes) onto the cluster you just provisioned.
```

Replace market-y, colloquial language with matter-of-fact doc-style language.

Reply:
Pushed this exact change to main.